micropython/senml/LICENSE

```diff
@@ -0,0 +1,22 @@
+MIT License
```

Comment:
Now that this license is copied at the top of each file, this LICENSE file itself can be removed (otherwise it's confusing why it's here, no other packages in this repo have one, I don't think).